src/org/zaproxy/zap/extension/exportreport/FrameExportReport.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ * This file is based on the Paros code file ReportLastScan.java
+ */
+package org.zaproxy.zap.extension.exportreport;
+
+import java.awt.CardLayout;
+import java.awt.Container;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JButton;
+import javax.swing.JPanel;
+import javax.swing.SpringLayout;
+import javax.swing.border.CompoundBorder;
+import org.parosproxy.paros.view.AbstractFrame;
+import org.parosproxy.paros.Constant;
+
+/*
+ * AUTHOR		:	GORAN SARENKAPA - JordanGS
+ * SPONSOR		:	RYERSON UNIVERSITY
+ */
+
+public class FrameExportReport extends AbstractFrame implements ActionListener
+{
+	private final String			FINISH				= Constant.messages.getString("exportreport.forward.finish.label");
+	private final String			NEXT				= Constant.messages.getString("exportreport.forward.next.label");
+	private ExtensionExportReport	extension			= null;
+	private JPanel					pnlCards;
+	private JButton					btnBack;
+	private JButton					btnNext;
+	private PanelNav				pnlNav;
+	private int						curCard				= 0;
+	final String					PANEL_SOURCE		= Constant.messages.getString("exportreport.panel.source.label");
```

Comment:
Yes it does. It is also the title for each panel visible to the user. Two birds with one stone :)